alerts/machine-config-operator/ExtremelyHighIndividualControlPlaneMemory.md

```diff
@@ -0,0 +1,27 @@
+# ExtremelyHighIndividualControlPlaneMemory
+
+## Meaning
+
+This alert fires when the memory usage per an instance within control plane
+nodes exceeds 90% of total memory available per a node.  
+This is a critical level alert.
+
+## Impact
+
+At such very low free memory, the cluster becomes unstable and expected slow
+responses from kube-apiserver or failing requests specially on etcd.
+Moreover, OOM kill is expected which negatively influences the pod scheduling.
```

Comment:
s/and expected slow responses from kube-apiserver/and you can expect slow response from the kube-apiserver
s/specially on etcd/especially from etcd
s/expected which/expected, which